src/Symfony/Component/Messenger/Middleware/MiddlewareInterface.php

```diff
@@ -23,11 +23,3 @@
      */
     public function handle(Envelope $envelope, callable $next): void;
```

Comment:
worth to describe the callable signature / different mode in the phpdoc?

Reply:
ah it's pretty clear in NextInterface, `@see NextInterface` maybe?

Reply:
that's what the `@param` annotation already links to to me

Reply:
sorry i missed that